packages/aether_observatory/Rakefile

```diff
@@ -5,7 +5,7 @@
 begin
   require "bundler/setup"
 rescue LoadError
-  puts "You must `gem install bundler` and `bundle install` to run rake tasks"
+  Rails.logger.debug "You must `gem install bundler` and `bundle install` to run rake tasks"
```

Comment:
Not sure about this change. Rails.logger can very likely be nil here, and Rails might have not been required at all depending on where bundler/setup failed.